src/common/ceph_crypto.cc

```diff
@@ -203,6 +204,21 @@ ssl::OpenSSLDigest::~OpenSSLDigest() {
   }
 }
 
+ssl::OpenSSLDigest::OpenSSLDigest(OpenSSLDigest&& o) noexcept
```

Comment:
you fixed a real howler here, well done

Reply:
i'm hoping that nobody's tried copying/moving these before